include/core/reactor_common.h

```diff
@@ -198,13 +198,10 @@ void _lf_trigger_shutdown_reactions(environment_t* env);
  * Create dummy events to be used as spacers in the event queue.
  * @param env Environment in which we are executing.
  * @param trigger The eventual event to be triggered.
- * @param time The logical time of that event.
- * @param next The event to place after the dummy events.
- * @param offset The number of dummy events to insert.
- * @return A pointer to the first dummy event.
+ * @param tag The tag of that event.
+ * @return A pointer to the dummy event.
  */
-event_t* _lf_create_dummy_events(environment_t* env, trigger_t* trigger, instant_t time, event_t* next,
-                                 microstep_t offset);
+event_t* _lf_create_dummy_events(environment_t* env, trigger_t* trigger, tag_t tag);
```

Comment:
Given that these dummy events have a difference use now, I suggest replacing the function description as follows:
```
 * @brief Create a dummy event with the specified tag.
 * 
 *  A dummy event is an event with no triggers that can be put on the event queue to trigger a tag advance to the specified tag.
```

Reply:
I replaced the comment as you suggested in [5c966b5](https://github.com/lf-lang/reactor-c/pull/390/commits/5c966b50a131053c738a93ffd7329b020277aa49).